.gitignore

```diff
@@ -8,3 +8,5 @@ releases/*.tar.gz
 releases/*.tar.gz.sha256sum
 _output/
 .vagrant/
+.idea/
```

Comment:
These are better set in your global .gitignore; I think we should keep the gitignore specific for files that this repository produces, and not take all possible editors into account.

Maybe we need to add a comment to describe that, similar to https://github.com/moby/moby/blob/7d30dafd69ed06e94a0e1b87f74a1f7c9a98dfeb/.gitignore#L1

Reply:
ACK. I add the comment in this PR or just close this one?

Reply:
Perhaps adding the comment wouldn't hurt, so if you want to update the PR 👍

(if you do, please amend the commit, so that there's only a single commit in the PR)

Reply:
updated.